src/components/SingleBook.js

```diff
@@ -0,0 +1,30 @@
+import './SingleBook.css';
+
+const SingleBook = () => (
```

Comment:
- [ ] `SIngleBook` component should take the book data as a prop and renders the title and the author